site/components/Why.tsx

```diff
@@ -1,42 +1,42 @@
-import { Card, Text, Heading, SimpleGrid, Stack, useColorModeValue } from "@chakra-ui/react"
+import { Card, Text, Heading, SimpleGrid, Stack, useColorModeValue } from '@chakra-ui/react'
 
 const WhyBlock = ({ title, children }) => {
-    return (
-        <Card background='bw.50'>
-            <Stack p={3}>
-                <Heading as='h3' size='md'>{title}</Heading>
-                <Text lineHeight={1.8} fontSize={'md'}>{children}</Text>
-            </Stack>
-        </Card>
-    )
+  return (
+    <Card background='bw.50'>
+      <Stack p={3}>
+        <Heading as='h3' size='md'>{title}</Heading>
+        <Text lineHeight={1.8} fontSize={'md'}>{children}</Text>
+      </Stack>
+    </Card>
+  )
 }
 
 const Why = () => {
-    return (
-        <Stack spacing='10'>
-            <Heading as='h2'>Why?</Heading>
-            <SimpleGrid columns={1} spacing={10} minChildWidth='300px'>
-                <WhyBlock title='Verifiable computation'>
+  return (
+    <Stack spacing='10'>
+      <Heading as='h2'>Why?</Heading>
+      <SimpleGrid columns={1} spacing={10} minChildWidth='300px'>
+        <WhyBlock title='Verifiable computation'>
                     Run computations and generate a proof that the computation was run as designed.
```

Comment:
My editor is still re-formatting this line?

Reply:
@calum I checked on my local copy of `explorerer` (from where the `eslint` rules were sourced), and here's what I found (note that I'm not very familiar with VSCode) which mirrors the behaviour of this PR:
  1. Without the `eslintConfig` section (or `eslint` in general), VSCode reformats on save (default behaviour, I presume since I'd not tweaked those on my VSCode instance).
  2. With the `eslintConfig` rules present, VSCode reformats on save again, but respects the `indent` parameter passed in (2, 4, etc.) **but** `eslint` does not mark the text nodes/entries as errors based on the indentation.
  3. So even if we were to manually set indents to some random spacing using some other plain text editor, `eslint` would not recognise that indentation as erroneous (and so `fix` would do nothing as well). However, when saving in VSCode, it would still auto-format the text nodes as well (according to the value of the "Spaces: ..." value shown in the bottom-right of the editor window).

tl;dr: So `eslint` by itself seems to be working correctly (not recognising text nodes), and I couldn't find any configuration for `eslint` relating to this.

VSCode seems to auto-format on save by default and it does so even for text nodes.